docs/docs/glossary.md

```diff
@@ -16,326 +16,329 @@ slug={props.slug}
 
 ### AST
 
-Abstract Syntax Tree: A tree representation of the source code that is found during a [compilation](#compiler) step between two languages. For example, [gatsby-transformer-remark](/packages/gatsby-transformer-remark/) will create an AST from [Markdown](#markdown) to describe a Markdown document in a tree structure using the [Remark](#remark) parser.
+Árvore de Sintaxe Abstrata: Uma representação em árvore do código fonte encontrado durante uma etapa de [compilação](#compiler) entre duas linguagens. Por exemplo, [gatsby-transformer-remark](/packages/gatsby-transformer-remark/) criará um AST do [Markdown](#markdown) para descrever um documento Markdown em uma estrutura de árvore usando o analisador [Remark](#remark).
 
 ### API
 
-Application Programming Interface: A method for one application to communicate with another. For example, a [source plugin](#source-plugin) will often use an API to get its data.
+Interface de Programação de Aplicações: Um método para um aplicativo se comunicar com outro. Por exemplo, um [plugin nativo](#source-plugin) geralmente usa uma API para obter seus dados.
 
-### Accessibility
+### Acessibilidade
 
-The inclusive practice of removing barriers that prevent interaction with, or access to websites, by people with disabilities. When sites are correctly designed, developed and edited for accessibility, generally all users have equal access to information and functionality. Read about [Gatsby's Commitment to Accessibility](/blog/2019-04-18-gatsby-commitment-to-accessibility/).
+A prática inclusiva de remover barreiras que impedem a interação ou acesso a sites por pessoas com deficiência. Quando os sites são projetados corretamente, desenvolvidos e editados para acessibilidade, geralmente todos os usuários têm acesso igual às informações e funcionalidades. Ler sobre [Compromisso do Gatsby com a Acessibilidade](/blog/2019-04-18-gatsby-commitment-to-accessibility/).
```

Comment:
Talvez:
Leia sobre o [Compromisso do Gatsby com a Acessibilidade](/blog/2019-04-18-gatsby-commitment-to-accessibility/).